File: GoLang/8 kyu/Twice as old/main.go
package main

import "fmt"

func main() {
	fmt.Println(TwiceAsOld(29, 7))
}

func TwiceAsOld(dadYearsOld, sonYearsOld int) int {
	for i := 0; i < 200; i++ {
	  if (dadYearsOld - i) == (sonYearsOld - i)*2 || (dadYearsOld + i) == (sonYearsOld + i)*2 {
		return i
	  }
	}
	return 0
  }

File: Python/8 kyu/Twice as old/main.py
def twice_as_old(dad_years_old, son_years_old):
    for n in range(200):
        if (dad_years_old - n) == 2*(son_years_old - n) or (dad_years_old + n) == 2*(son_years_old + n):
            return n

if __name__ == "__main__":
    print(twice_as_old(29, 7))